src/lib/Form/Data/FormMapper/ContentTypeDraftMapper.php

```diff
@@ -80,10 +91,25 @@ public function mapToFormData(ValueObject $contentTypeDraft, array $params = [])
             $contentTypeData->descriptions[$language->languageCode] = $contentTypeDraft->getDescription($baseLanguage->languageCode);
         }
 
+        $metaFieldTypeIdentifiers = $this->contentTypeFieldTypesResolver->getMetaFieldTypeIdentifiers();
+
+        try {
+            $contentType = $this->contentTypeService->loadContentType($contentTypeDraft->id);
+        } catch (NotFoundException $exception) {
+            $contentType = null;
+        }
+
         foreach ($contentTypeDraft->fieldDefinitions as $fieldDef) {
+            $isMetaFieldType = in_array($fieldDef->fieldTypeIdentifier, $metaFieldTypeIdentifiers, true);
+
+            $enabled = $isMetaFieldType
+                && null !== $contentType
+                && null !== $contentType->getFieldDefinition($fieldDef->identifier);
```

Comment:
Why not
```suggestion
                && null !== $contentType->hasFieldDefinition($fieldDef->identifier);
```
?

Reply:
I'm ok with this but condition should be `$contentType->hasFieldDefinition($fieldDef->identifier)`